src/main/java/core/basesyntax/service/operation/FruitService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.service.db.Storage;
+import core.basesyntax.service.model.FruitTransaction;
+
+public interface FruitService {
```

Comment:
maybe not service - hendler? And move it to strategy package because this logic refers to the pattern strategy